src/utils/SqsClient.ts

```diff
@@ -15,4 +15,4 @@ const sqsClientRaw = new SQSClient({
 
 const sqsClient = process.env.XRAY_ENABLED === "true" ? AWSXRay.captureAWSv3Client(sqsClientRaw as any) : sqsClientRaw;
 
-export { sqsClient };
+export { sqsClient, SendMessageCommand };
```

Comment:
is SendMessageCommand being used?

Reply:
I removed from the import in CICService and added to sqsClient in Utils, it's being used here https://github.com/alphagov/di-ipv-cri-cic-api/blob/main/src/services/CicService.ts#L11